web/src/features/fireWeather/components/tables/SortableTableByDatetime.tsx

```diff
@@ -72,68 +82,98 @@ function SortableTableByDatetime<R extends WeatherValue>(props: Props<R>) {
     setOrder(order === 'asc' ? 'desc' : 'asc')
   }
 
+  interface TableHeaderProps {
+    title: string
+    testId?: string
+  }
+
+  const TableHeader = (tableHeaderProps: TableHeaderProps) => {
+    const tableHeaderClasses = useStyles()
+    return (
+      <Toolbar data-testid={`${tableHeaderProps.testId}-header`} disableGutters={true}>
```

Comment:
Toolbar is the one that has min height that takes up an extra room (discussion earlier). I don't see why we need this can you remove it?